### DISCUSSION TITLE
Command output?

### DISCUSSION THREAD

**Discussion by antrobot1234** [Pryaxis/TShock#1737] [General]
is there any way i can run commands either as superadmin or via the server console where the command isn't announced in chat? i tried both the regular / and the "silent" . but both were announced. I don't want my commands to be announced like that all the time.

**Comment by QuiCM** [comment]
Unfortunately the 'silence' command prefix depends on the author of the command supporting it. I.E., if they haven't written code to support silent usage, it won't be silent :(
